test/TestUtilities/Test.Utility/Protocol/TestContent.cs

```diff
@@ -11,15 +13,20 @@ namespace Test.Utility
 {
     public class TestContent : HttpContent
     {
-        private MemoryStream _stream;
+#pragma warning disable CA2213
+        // Need to disable this rule because the analyzer thinks We are not disposing _stream
+        // when in reality, it is
```

Comment:
Should we link to the roslyn tracking issue, so anyone looking at this comment in the future can track if it's fixed yet, and remove the warning suppression?

Reply:
Added related roslyn issue. Thanks!